switchtheme.module

```diff
@@ -187,6 +201,34 @@ function switchtheme_block_view($delta = '') {
   }
 }
 
+/**
+ * returns an unordered list of active themes to switch to.
+ */
+
+function switchtheme_return_itemlist_block() {
+  $themes = list_themes();
+  $enabledThemes = []; 
+
+  $current_url = url(current_path(), array('absolute' => TRUE));
+  foreach ($themes as $key => $theme) {
+    if ($theme->status) { // if enabled
+      $items[] = array(
```

Comment:
You are defining the `$enabled_themes` array variable earlier (well, `$enabledThemes` but that is not acceptable according to our PHP coding standards), but you never use it. I am assuming that it's meant to be used here 🙂 

```suggestion
      $enabled_themes[] = array(
```